.travis.yml

```diff
@@ -8,19 +7,14 @@ services:
   - postgresql
 addons:
   postgresql: '9.5'
-  apt:
-    packages:
-    - python3
-    - python3-pip
-    - python3.4-venv
 cache:
   directories:
   - $HOME/.gradle/caches
   - $HOME/.gradle/wrapper
   - $HOME/.m2
 before_script:
 # work around https://github.com/travis-ci/travis-ci/issues/8363
-- pyenv global system 3.5
+- pyenv global system 3.6
```

Comment:
Why is this needed if you use python3's venv?

Reply:
`pyenv` sets the python version used, as there are multiple versions installed on the Travis CI image and 3.6 is not the default. Please see also discussion in https://github.com/travis-ci/travis-ci/issues/8363